client/src/components/App/AppToolbar/AppToolbar.tsx

```diff
@@ -8,16 +8,20 @@ import IsActiveToggle from 'commons/IsActiveToggle/IsActiveToggle';
 import { landingPageRoute, usersManagementRoute } from 'Utils/Routes/Routes';
 
 import useAppToolbar from './useAppToolbar';
+import useStyles, { AppToolbarClasses } from './AppToolbarStyles';
 
 const toggleMessage = 'מה הסטטוס שלך?';
 const navButtonsWhitelist = {
   allowedUserTypes: [UserType.ADMIN, UserType.SUPER_ADMIN],
   allowedRoutes: [landingPageRoute, usersManagementRoute]
 };
 
-const StatePersistentNavLink = (props: NavLinkProps) => {
+interface StatePersistentNavLinkProps extends NavLinkProps {
+  classes: AppToolbarClasses;
+}
+
+const StatePersistentNavLink = (props: StatePersistentNavLinkProps) => {
```

Comment:
classes dont have to be passes through the `useAppToolbar` custom hooks since const classes = useStyles(); can just be called

Reply:
fixed with 3 commits too many